### ISSUE TITLE
Switch to 3-clause BSD

### ISSUE THREAD

**Issue by rhattersley** [SciTools/libmo_unpack#12]

**Comment by rhattersley** [comment]
NB. This may be complicated by the inclusion of code from HDF (e.g. convert_float_ibm_to_ieee32.c).

**Comment by marqh** [comment]
we should check, but I believe that:
https://github.com/SciTools/libmo_unpack/blob/master/src/convert_float_ibm_to_ieee32.c

```
NCSA HDF Version 3.10 source code and documentation are in the public         
* domain.  Specifically, we give to the public domain all rights for future                                                                          
* licensing of the source code, all resale rights, and all publishing rights.                                                                         
*                                                                               
* We ask, but do not require, that the following message be included in all                                                                             
* derived works:       
...
```

enables us to relicense and include their message in the header at will

**Comment by pelson** [comment]
CLosed by #15
